dlt/destinations/impl/sink/sink.py

```diff
@@ -193,23 +144,21 @@ def update_stored_schema(
         return super().update_stored_schema(only_tables, expected_update)
 
     def start_file_load(self, table: TTableSchema, file_path: str, load_id: str) -> LoadJob:
+        load_state = destination_state().setdefault(load_id, {})
```

Comment:
set default is not thread safe. but there won't be anyone else accessing this key so OK